Apps/AdminWebClient/src/ClientApp/src/services/restCommunicationService.ts

```diff
@@ -37,4 +37,30 @@ export class RestCommunicationService implements ICommunicationService {
         });
     });
   }
+
+  public getBannerList(): Promise<Communication[]> {
+    return new Promise((resolve, reject) => {
+      this.http
+        .get<RequestResult<Communication[]>>(`${this.BASE_URI}`)
+        .then(requestResult => {
+          this.handleResult(requestResult, resolve, reject);
+        })
+        .catch(err => {
+          console.log(err);
+          return reject(err);
+        });
+    });
+  }
+
+  private handleResult(
+    requestResult: RequestResult<any>,
+    resolve: any,
+    reject: any
+  ) {
+    if (requestResult.resultStatus === ResultType.Success) {
```

Comment:
Do you want Identity here or Equality?  I think equality...

Reply:
Looks like we use both equality and identity for vue service functions in different places throughout the app. Shouldn't cause any problems but we should commit to using one or the other for consistency.

Reply:
Right, I want to use strict equality. the === operator will not perform type conversion and I believe it will provide better performance than the ==.
May we suggest to use === instead of == everywhere if possible? Unless in some exceptional cases, we may need to use ==?

Reply:
+1